docs/guide/review.md

```diff
@@ -0,0 +1,85 @@
+`tsrc` can automate part of the code review process.
+
+You can use it with GitLab or GitHub repositories.
+
+# Handling GitLab merge requests
+
+## Configuration
+
+The first step is to log in to GitLab and get your personal access token.
+
+Then, write a file in `~/.config/tsrc.yml` containing the token:
+
+```yaml
+auth:
+  gitlab:
+    token: <your token>
+```
+
+!!! note
+    The full config file format is described in the [reference](../ref/formats.md).
+
+The second step is to tell `tsrc` about the HTTP url of your GitLab instance, which is needed to call the [GitLab HTTP API](https://docs.gitlab.com/ce/api/) (currently using version *4*).
+
+This is done in the *manifest* file:
+
+```yaml
+gitlab:
+  url: http://gitlab.local
+
+repos:
+ - ...
+```
+
+## Creating and accepting merge requests
+
+Here's how you can create and assign a merge request:
+
+```bash
+# start working on your branch
+$ tsrc push [--assignee ASSIGNEE]
+```
+
+
+When the review is done, you can accept it and let GitLab merge the branch once
+the CI pipeline passes with the following command:
+
+```bash
+$ tsrc push --accept
+```
+
+# Handling GitHub pull requests
+
+If there is a remote named `origin` and starting with `git@github.com`, `tsrc` will assume you want to use GitHub.
+
+Then, the first time you need access to GitHub API, it will ask for your credentials, generate a token and store it in the `~/.config/tsrc.yml` file.
+
+## Creating a pull request
+
+Here's how to create a pull request and request reviewers:
+
+```bash
+# start working on your branch
+$ tsrc push [--reviewer REVIEWER ...] [--assignee ASSIGNEE}
```

Comment:
} should be ] at the end of the line

Reply:
Also `...` could suggest it is ok to have `tsrc push --reviewer R1 R2` instead of `tsrc push --reviewer R1 --reviewer R2`.

So I would remove `...` and stick to `tsrc push [--reviewer REVIEWER] [--assignee ASSIGNEE]`.

Your comment below makes it very clear that you can have several reviewers.